src/API/Endpoints/Reports/RecentDonations.php

```diff
@@ -16,13 +16,19 @@ public function __construct() {
 
 	public function get_report( $request ) {
 		// Setup donation query args (get sanitized start/end date from request)
+
+		$gateways = give_get_payment_gateways();
```

Comment:
@henryholtgeerts I see we are using the same code at multiple places can we create a function `protected` function to prevent code repetition.

Reply:
Refactored endpoints to all use the "get_payments" function. Ref: https://github.com/impress-org/givewp/commit/96fdfd70585046f30e3b5c0ec6de03d21abccbf8